libp2p/Session.cpp

```diff
@@ -406,7 +407,9 @@ bool Session::checkRead(std::size_t _expected, boost::system::error_code _ec, st
     {
         // with static m_data-sized buffer this shouldn't happen unless there's a regression
         // sec recommends checking anyways (instead of assert)
-        cnetlog << "Error reading - TCP read buffer length differs from expected frame size.";
+        LOG(m_netLoggerError)
+            << "Error reading - TCP read buffer length differs from expected frame size ("
```

Comment:
Also seems too low-level for ERROR

Reply:
@gumb0 The comment above the log statement gives me the impression that this should never happen which is why I chose to log this as error (since if it does happen we want to know about it).